facade/src/main/java/org/jboss/pnc/facade/providers/GroupBuildProviderImpl.java

```diff
@@ -130,18 +142,23 @@ private Consumer<Result> notifyOnDeletionCompletion(String callbackUrl) {
     }
 
     @Override
-    public Page<GroupBuild> getGroupBuilds(
-            int pageIndex,
-            int pageSize,
-            String sort,
-            String q,
-            String groupConfigurationId) {
-        return queryForCollection(
-                pageIndex,
-                pageSize,
-                sort,
-                q,
-                withBuildConfigSetId(Integer.valueOf(groupConfigurationId)));
+    public Page<GroupBuild> getGroupBuilds(GroupBuildPageInfo groupBuildPageInfo, String groupConfigId) {
+        Integer groupConfigIdModel = groupConfigurationMapper.getIdMapper().toEntity(groupConfigId);
```

Comment:
was struggling a little bit with variable names in this part, so feel free to suggest better ones if it comes to your mind

Reply:
it's fine